### PR TITLE
Web: Fix TypeScript problems on main

### PR DESCRIPTION
**Pull request by vovakulikov**
It seems like we missed typescript problems a few times in story files. This PR just fixes ts problems on the main. 

## Test plan
- Check that CI is green and doesn't have problems with eslint/ts builds

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```
Comment (eseliger) [inline pnpm-lock.yaml:5]: is this also part of the fix?

File: client/web/src/cody/update/UpdateInfoContent.story.tsx [hunk 2]
```diff
@@ -114,7 +114,7 @@ export const CheckNow: Story = {
 
 export const ReleaseDetails: Story = {
     render: function Render() {
-        const [state, setState] = useState<UpdateInfo>({
+        const [state] = useState<UpdateInfo>({
```
Comment (eseliger) [inline client/web/src/cody/update/UpdateInfoContent.story.tsx:117]: I wonder what we need this useState for at all, but maybe it has a use?

**Review by eseliger — APPROVED** [review]
<3

File: client/web/src/cody/update/UpdateInfoContent.story.tsx (hunk 2 above)
Comment (philipp-spiess) [reply client/web/src/cody/update/UpdateInfoContent.story.tsx:117]: Yeah we can probably pull the object outside of the function scope to make it stable too!

**Review by philipp-spiess — APPROVED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (vovakulikov) [reply pnpm-lock.yaml:5]: My bad, locally I have a different version of pnpm, we probably should update it in our package.json file but this is for other PR I think. Reverted these changes

**Pull request merged**